c2_utils/src/mfx_va_frame_pool_allocator.cpp

```diff
@@ -86,15 +102,15 @@ mfxStatus MfxVaFramePoolAllocator::AllocFrames(mfxFrameAllocRequest *request,
 
             response->NumFrameActual = 0;
 #define RETRY_TIMES 5
-            for (int i = 0; i < max_buffers; ++i) {
+            for (int i = 0; i < opt_buffers; ++i) {
 
                 std::shared_ptr<C2GraphicBlock> new_block;
                 int retry_time_left = RETRY_TIMES;
                 do {
                     res = m_c2Allocator->fetchGraphicBlock(
                         request->Info.Width, request->Info.Height,
                         MfxFourCCToGralloc(request->Info.FourCC),
-                        { C2AndroidMemoryUsage::CPU_READ|C2AndroidMemoryUsage::HW_COMPOSER_READ, C2AndroidMemoryUsage::HW_CODEC_WRITE },
+                        { m_consumerUsage, C2AndroidMemoryUsage::HW_CODEC_WRITE },
```

Comment:
Is there somewhere to change m_consumerUsage? Except default 0.
I didn't find anywhere call the function SetConsumerUsage.